### PR TITLE
feat: `bring vite`

### PR DESCRIPTION
**Pull request by skyrpex**

### CODE REVIEW

File: .github/workflows/vite-pull.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        with:
+          node-version: 20.x
+          registry-url: https://registry.npmjs.org
+      - name: Install winglang
+        run: npm i -g winglang
+      - name: Install dependencies
+        run: npm install
+        working-directory: vite
+      - name: Test
+        run: wing test **/*.test.w
```
Comment (MarkMcCulloh) [inline .github/workflows/vite-pull.yaml:25]: This isn't needed, `wing test` will grab all tests in the directory for you

```suggestion
        run: wing test
```

Comment (skyrpex) [reply .github/workflows/vite-pull.yaml]: Thanks! I didn't think of changing the script too much.

File: vite/vite.cjs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  });
+
+  return new Promise((resolve) => {
+    child.stdout.on("data", (chunk) => {
+      const payload = parsePayload(chunk.toString());
+      if (payload) {
+        resolve(payload.url);
+        child.stdout.removeAllListeners("data");
+      } else {
+        // console.log(chunk.toString());
```
Comment (ainvoner) [inline vite/vite.cjs:49]: do we need it?

**Review by ainvoner — APPROVED** [review]

File: vite/vite.cjs (hunk 2 above)
Comment (skyrpex) [reply vite/vite.cjs]: Not really, although I think we should have a way to log whatever Vite is outputting somewhere. I had an issue in Wing to discuss this, somewhere. I'll look for it.

**Pull request merged**